docs/development/building/Building-in-Fedora.md

```
@@ -1,41 +1,98 @@
-# Building in Fedora 35
+---
+sidebar_position: 4
+sidebar_label: Building in Fedora
+title: Building in Fedora
+---
+
+# Building in Fedora 44
+
+This guide covers building the Betaflight firmware and configurator on Fedora 44.
+
+### Clone Betaflight Repository and Install Toolchain
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)